<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>657</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>8000</width>
    <height>8000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QToolBox" name="tab1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#frameMisNotas{
	border: 1px solid lightgray;
	width: 100%;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="tabSpacing">
       <number>6</number>
      </property>
      <widget class="QWidget" name="tabpage1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>842</width>
         <height>494</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Notas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_tituloVentana">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">max-height:25px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="text">
           <string>Mis Notas</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>-2</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frameMisNotas">
          <property name="styleSheet">
           <string notr="true">max-height: 440px;
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="2">
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="1" column="0">
               <widget class="QPushButton" name="btn_creadas">
                <property name="text">
                 <string>Notas Creadas</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPlainTextEdit" name="txt_mensaje">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>202</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">height:200px;</string>
                </property>
                <property name="plainText">
                 <string>Seminario De Lenguajes I


Integrantes:

Gutierrez Garcia Josias
Taritolai Nicolas Daniel
Aramayo Enzo Demian
	</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="btn_archivadas">
                <property name="text">
                 <string>Notas Archivadas</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_tituloNota">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Ingrese el titulo:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLineEdit" name="txt_tituloNota">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lbl_contenidoNota">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Ingrese el contenido:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QPlainTextEdit" name="ptxt_contenidoNota">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="btn_guardar">
               <property name="font">
                <font>
                 <family>Bahnschrift Light SemiCondensed</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Guardar</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="btn_cancelar">
               <property name="font">
                <font>
                 <family>Bahnschrift Light SemiCondensed</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cancelar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabpage2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>842</width>
         <height>420</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Creadas</string>
       </attribute>
       <widget class="QPushButton" name="btn_modCreadas">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>130</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_archivarCreadas">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>190</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Archivar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_eliminarCreadas">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_notasCreadas">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>351</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Notas Creadas</string>
        </property>
       </widget>
       <widget class="QListWidget" name="wdglist_notasCreadas">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>541</width>
          <height>571</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>30</y>
          <width>231</width>
          <height>691</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>60</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_archivadas_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Light SemiCondensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Notas Archivadas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_creadas_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Light SemiCondensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Notas Creadas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txt_mensaje_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="plainText">
            <string>Seminario De Lenguajes I


Integrantes:

Gutierrez Garcia Josias
Taritolai Nicolas Daniel
Aramayo Enzo Demian
	</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabpage3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>842</width>
         <height>420</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Archivadas</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>351</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Notas Archivadas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>230</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_tareas_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>581</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Archivar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>130</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>90</y>
          <width>231</width>
          <height>731</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>60</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_archivadas_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Light SemiCondensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Notas Archivadas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_creadas_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Light SemiCondensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Notas Creadas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txt_mensaje_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="plainText">
            <string>Seminario De Lenguajes I


Integrantes:

Gutierrez Garcia Josias
Taritolai Nicolas Daniel
Aramayo Enzo Demian
	</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabpage4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>842</width>
         <height>420</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Tareas</string>
       </attribute>
       <widget class="QProgressBar" name="bar_progresoTotal">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>750</y>
          <width>1031</width>
          <height>31</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>370</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_ingresarTarea">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>550</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>400</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>340</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ingresarTarea">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>550</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ingresar Tarea</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>710</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Progreso Total</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_vaciarLista_3">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>80</y>
          <width>141</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vaciar Lista</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>561</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
